include/kyosu/details/hyperg/hyp2_1/hyp2_1_base.hpp

```diff
@@ -14,28 +14,31 @@
 namespace kyosu::_
 {
```

Comment:
There is a bunch of // code in this files, too much to list them all. Please carefully remove them from every places

Reply:
done